lib.php

```diff
@@ -144,6 +144,11 @@ function populate_zoom_from_response(stdClass $zoom, stdClass $response) {
             $newzoom->$field = $response->$field;
         }
     }
+
+    if (isset($response->encrypted_password)) {
+        $newzoom->join_url = $newzoom->join_url . '?pwd=' . $response->encrypted_password;
```

Comment:
@Naveen1895 I believe this will need the check for ?pwd existance check and then replace. Plese correct me if I'm wrong.

Reply:
Yes, correct! I have fixed that and pushed. could you please verify